app/models/follow_up.rb

```diff
@@ -2,34 +2,24 @@
 
 require 'rest-client'
 
-class FollowUp
-  include ActiveModel::Validations
-
-  attr_accessor :email, :name, :language, :destination
+class FollowUp < ActiveRecord::Base
+  belongs_to :language
+  belongs_to :destination
 
   validates :email, presence: true, email_format: { message: 'Invalid email address' }
   validates :language, presence: true
   validates :destination, presence: true
 
-  def initialize(email, language_id, destination_id, name = nil)
-    self.email = email
-    self.language = Language.find(language_id)
-    self.destination = Destination.find(destination_id)
-    self.name = name
-  end
-
   def send_to_api
-    validate_fields
+    save!
```

Comment:
This will result in `save!` being called immediately after a `create!`. I can't remember how smart rails is about this these days. If this results in an extra sql update call, you'll want to add a check here like:
```ruby
save! if changed?
```